forms-flow-web/src/components/ServiceFlow/list/sort/CreateNewFilter.js

```diff
@@ -544,18 +545,30 @@ export default function CreateNewFilterDrawer({
     }));
   }, [userList]);
 
+  const removeTenantKey = (value, tenantkey) => {
+    const tenantKeyCheck = value.match(`${tenantkey}-`);
+    if (
+      tenantKeyCheck &&
```

Comment:
please add tenantKeyCheck.length check also. If there is no data in tenantKeyCheck array, current condition will break